yudien/data/udn_test_cases/strconcat/input.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Seems like the function for the test shouldnt be "strconcat" but "__concat", so that it matches the actual function.

Or it can match the UDN_StrConcat.  It should match either the Go functions or the UDN label, right now it matches nothing.  So this cant have automation applied to it well.

Reply:
Ah, the `strconcat` is just a directory name -- it actually has nothing to do with the test run other than the name printed during the test run.